### DISCUSSION TITLE
预言：感谢党和政府，警惕境外势力利用

### DISCUSSION THREAD

**Discussion by allenyan513** [WeNeedHome/SummaryOfLoanSuspension#623] [General]
预言：感谢党和政府，警惕境外势力利用
感谢党和政府帮助我们解决了烂尾楼停工问题，但有极个别别有用心的境外势力利用此次事件，将停贷信息汇总到了境外Github网站中，企图为境外势力递刀子，对此与我们无关。

**Comment by jonny77** [comment]
发在国内有人删帖,倒是给个地方说话啊.

**Reply by Crazy-Creeper** [reply]
老爷们不爱听

**Comment by Accelerator6666** [comment]
合着你买的楼盘复工了，别人的烂尾楼就是给别人递刀子了是吗？

**Comment by taopeng-life** [comment]
一辈子积蓄搭进去了，我管它是给谁递刀子，要是洋大人能利用这个，让我楼盘复工我真是谢谢它

**Comment by liuwenwu520** [comment]
这个是个钓鱼贴

**Reply by taopeng-life** [reply]
管他是不是钓鱼贴，说这种话就是找骂

**Comment by jiushi555** [comment]
解决？了！

**Comment by TALK1949** [comment]
我怎么看你们倒很想把这件事跟境外势力、反华扯上关系？
自己骂自己很开心？
